PT-164804897 GC of aec_signed_tx objects (#2328)
aeternity/aeternity: docs/release-notes/next/PT-164804897-garbage-collect-aec_signed_tx.md

> * Introduces new database garbage collection of aec_signed_tx objects.

Reviewer: A bit more user oriented?
```suggestion
* Introduces garbage collection of transactions from the database.
```
